test/index.js

```diff
@@ -0,0 +1,18 @@
+var R = require('..');
+var fs = require('fs');
+var path = require('path');
+var eq = require('./shared/eq');
+
+
+describe('api surface', function() {
+  var srcFolder = path.join(__dirname, '../src');
+  var isJsFile = function(file) { return file.match(/\.js$/); }
+  var removeJsEnding = function(file) { return file.replace('.js', ''); }
+
+  var exportedApi = Object.keys(R);
+  var actualApi = fs.readdirSync(srcFolder).filter(isJsFile).map(removeJsEnding);
```

Comment:
What is the purpose of the test? I would say it's to verify that `require('ramda/index.js')` and `require('ramda/dist/ramda.js')` are equivalent. Note that I did not mention the locations of files other than __index.js__ and __dist/ramda.js__.

The test you've provided asserts this equivalence, but indirectly. It forces us to make assumptions about the project layout. Were we to add __src/somePrivateFunction.js__ our assertion would fail because `R` would not have a property named `somePrivateFunction`.